.gitignore

```diff
@@ -2,6 +2,8 @@
 # Created by https://www.gitignore.io/api/c,tex,c++,cmake,jetbrains+all
 # Edit at https://www.gitignore.io/?templates=c,tex,c++,cmake,jetbrains+all
 
+build
```

Comment:
Why do we need build directory in gitignore? We anyway cannot build zk without it's directories.

Reply:
By mistake, removed.